knowrob_common/knowrob_common/src/main/java/org/knowrob/prolog/PrologInterface.java

```diff
@@ -62,17 +64,17 @@ public static void initJPLProlog(String initPackage) {
     	if(!initialized) {
 
     		try {
-    			Vector<String> args= new Vector<String>(Arrays.asList(jpl.fli.Prolog.get_default_init_args()));
+    			Vector<String> args= new Vector<String>(Arrays.asList(org.jpl7.fli.Prolog.get_default_init_args()));
     			//            args.add( "-G256M" );
     			//args.add( "-q" );
     			args.add( "-nosignals" );
 
     			String rosprolog = RosUtilities.rospackFind("rosprolog");
-    			jpl.fli.Prolog.set_default_init_args( args.toArray( new String[0] ) );
+    			org.jpl7.fli.Prolog.set_default_init_args( args.toArray( new String[0] ) );
 
     			// load the appropriate startup file for this context
-    			new jpl.Query("ensure_loaded('"+rosprolog+"/prolog/init.pl"+"')").oneSolution();
-    			new jpl.Query("register_ros_package('"+initPackage+"')").oneSolution();
+    			new org.jpl7.Query("ensure_loaded('"+rosprolog+"/prolog/init.pl"+"')").oneSolution();
```

Comment:
can't you use just Query here? org.jpl7.Query imported at the top.
same for some other statements.